packages/components-native/src/Button/Button.tsx

```diff
@@ -81,7 +81,7 @@ interface CommonButtonProps {
   /**
    * **Use at your own risk:** Custom style for specific elements. This should only be used as a
    * **last resort**. Using this may result in unexpected side effects.
-   * More information [here](https://atlantis.getjobber.com/storybook/?path=/docs/guides-customizing-components--docs#unsafe_-props).
+   * More information in the Customizing components Guide.
```

Comment:
I think it would still be valuable to add the link in case the doc is viewed in Storybook.
```suggestion
   * More information in the [Customizing components Guide](https://atlantis.getjobber.com/guides/customizing-components).
```

However, a more general question, should I be seeing `UNSAFE_` prop docs on component pages on the new site? Because I don't see them for `Button` or `Text` on the new docs site.
<img width="330" alt="image" src="https://github.com/user-attachments/assets/955fd419-8146-41ed-9b4f-113089bcfe69" />

Reply:
Ok that's fair - I brought the links back in Text and Button.

It's strange you don't see the `UNSAFE_` props, because they are on prod (since these props were added previous to this PR)
![Screenshot 2025-02-24 at 3 02 28 PM](https://github.com/user-attachments/assets/b3e9bf3f-d24e-48a9-8e63-3181035277ea)

and here it is on local:
![Screenshot 2025-02-24 at 3 03 40 PM](https://github.com/user-attachments/assets/ff93732b-2533-4443-9e4f-43e2b54afe3e)

Reply:
Yup - since `UNSAFE` is only in the `Text` and `Button` mobile components right now, it'll only be visible in the mobile tab